spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/consumer/list.html

```diff
@@ -87,7 +87,7 @@ <h2 th:text="#{admin.api-keys-consumer.subtitle}">[Post Administration</h2>
                                 </thead>
                                 <tbody>
                                     <tr th:each="apiKey : ${apiKeys}" th:id="|@{item}${apiKey.id}|">
-                                        <td class="a-center " th:text=" ${apiKey.serviceApiName != null} ? ${apiKey.serviceApiName} : '' ">Service API name</td>
+                                        <td class="a-center " th:text=" (${apiKey.serviceApiName != null} ? ${apiKey.serviceApiName} : '') + (${apiKey.serviceApi != null} ? ' (' + ${apiKey.serviceApi.uriIdentifier} + ')' : '') ">Service API name</td>
```

Comment:
Just one null check of  `apiKey.serviceApi` should be sufficient, because if the `serviceApi` is existing, then you will be able to get `apiKey.serviceApi.name` and `apiKey.serviceApi.uriIdentifier`